modules/aws/ebs/variables.tf

```diff
@@ -0,0 +1,45 @@
+variable "instances" {
+  description = "List of instance IDs"
+  type        = "list"
+}
+
+variable "num_instances" {
+  description = "Number of instances"
+}
+
+variable "num" {
```

Comment:
I see you have sizes and types variable, it seems like you may be able to determine the num based on these two variables. We just need to confirm that you will not run into this https://github.com/hashicorp/terraform/issues/17421

Reply:
@bernadinm i hit exactly this limitation. Will add a comment about this limitation.